env.go

```diff
@@ -75,9 +75,10 @@ const (
 
 // Config for a Hermit environment.
 type Config struct {
-	Envars    envars.Envars `hcl:"env,optional" help:"Extra environment variables."`
-	Sources   []string      `hcl:"sources,optional" help:"Package manifest sources."`
-	ManageGit bool          `hcl:"manage-git,optional" default:"true" help:"Whether Hermit should automatically 'git add' new packages."`
+	Envars      envars.Envars `hcl:"env,optional" help:"Extra environment variables."`
+	Sources     []string      `hcl:"sources,optional" help:"Package manifest sources."`
+	ManageGit   bool          `hcl:"manage-git,optional" default:"true" help:"Whether Hermit should automatically 'git add' new packages."`
+	AddIJPlugin bool          `hcl:"idea,optional" default:"false" help:"Whether Hermit should automatically add the IntelliJ IDEA plugin."`
```

Comment:
I don't think this one needs to exist as it's only used once at `init` time, which is too late for this to apply.

Reply:
How can `Init` get the configuration option if not from `Config`?

Reply:
Yep true!